src/main/java/com/spotify/confidence/ConfidenceTypeMapper.java

```diff
@@ -0,0 +1,91 @@
+package com.spotify.confidence;
+
+import com.google.protobuf.Struct;
+import com.spotify.confidence.shaded.flags.types.v1.FlagSchema;
+import com.spotify.confidence.shaded.flags.types.v1.FlagSchema.SchemaTypeCase;
+import com.spotify.confidence.shaded.flags.types.v1.FlagSchema.StructFlagSchema;
+import dev.openfeature.sdk.exceptions.ParseError;
```

Comment:
We should split the exceptions as well (keep our own exception types).

Reply:
When splitting this, make sure that the ConfidenceFeatureProvider catches _our_ types and maps them to OF exceptions.